<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmationDialog</class>
 <widget class="QWidget" name="ConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_reject">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #73AF21 stop: 1 #598A1B);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #73AF21 stop: 1 #598A1B);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 14px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
	border-radius: 14px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
  border-radius: 14px;
}

QPushButton:disabled {
	color: rgb(143,152,160);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	border: 1px solid rgba(0, 0, 0, 0);
	border-radius: 14px;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>75</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 14px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 14px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 14px;
}</string>
   </property>
   <property name="text">
    <string>Terminate</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Terminate training process?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
